app/serializers/api/v3/statement_serializer.rb

```diff
@@ -0,0 +1,17 @@
+module API
+  module V3
+    class StatementSerializer < Blueprinter::Base
```

Comment:
The JSON response is not matching the existing [statements endpoint response from ECF](https://manage-training-for-early-career-teachers.education.gov.uk/api-reference/reference-v3.html#api-v3-statements-id-get-responses-examples):

```JSON
{
  "data": {
    "id": "cd3a12347-7308-4879-942a-c4a70ced400a",
    "type": "statement",
    "attributes": {
      "month": "May",
      "year": "2022",
      "type": "ecf",
      "cohort": "2021",
      "cut_off_date": "2022-04-30",
      "payment_date": "2022-05-25",
      "paid": true,
      "created_at": "2021-05-31T02:22:32.000Z",
      "updated_at": "2021-05-31T02:22:32.000Z"
    }
  }
}
```

Reply:
Its close, but we still need to wrap it inside `data`, current response:
```JSON
{
  "type": "statement"
  "id": 1,
  "attributes": {
    "cohort": 2023,
    "created_at": "2024-03-07T18:22:51Z",
    "cut_off_date": "2024-04-05",
    "month": "July",
    "paid": false,
    "payment_date": null,
    "updated_at": "2024-03-07T18:22:51Z",
    "year": 2023
  },
}
```

Reply:
Looking at the docs I think you apply the outer document when calling the renderer, so to wrap the above in `data:` you'd need to remember to do something like:

```ruby
API::V3::StatementSerializer.new(statement, root: "data")
```

Otherwise, you need a third serialiser (which is simple and sharable, but kind of ugly).

JBuilder is more flexible here by design.

Reply:
Also, `cohort` and `year` are expected to be string.